packages/jsii-calc/test/assembly.jsii

```diff
@@ -7,6 +7,9 @@
     ],
     "url": "https://aws.amazon.com"
   },
+  "bin": {
+    "calc": "bin/calc"
```

Comment:
Might be interesting to rename `bin/calc` to something else to remove the "coincidence" that it matches the script's name (the key here).

For example, you could call it `bin/run` instead.

There's also a chance this does not test fine on Windows (because Windows is hard 🤓). In such cases you'd need to drop in a CMD entry point (I'll give you more info on this once we know this is necessary, hopefully it won't be).

Reply:
Good idea.